sapphire/utils.py

```diff
@@ -20,7 +20,7 @@
 ERR = [-1, -999]
 
 #: Speed of light in vacuum in m / ns.
-c = 0.299792458
```

Comment:
No!

This might break `PEP8` but IMHO we cannot use `C` for the speed of light. 

See for example:
http://docs.astropy.org/en/stable/constants/